### PR TITLE
Pathfinder & More

### PR DESCRIPTION
**Pull request by Kesiath**
## About the PR
Introduces the Pathfinder, a small decommissioned military vessel capable of expeditions. Additionally, this PR includes Sprinter fixes and the new crewing system for the Anchor.

**Media**
image

**Changelog**
:cl: Kesiath
- add: Added the Pathfinder, a small expedition capable vessel.
- tweak: Balanced the Sprinter further based on community feedback.
- tweak: Changed Anchor crew settings and added the new console system for picking crew.

### CODE REVIEW

**Comment by dvir001** [comment]
"a **small** expedition capable vessel"
Amm

**Comment by Kesiath** [comment]
> "a **small** expedition capable vessel" Amm

While it seems a bit large, it's right within the small parameters. It ended up a bit longer then smalls usually allow, but the total tile count is right near the edge of what is allowed as a small. We do need to add the micro category for the really tiny ships, that would help differentiate it.

File: Resources/Prototypes/_NF/Shipyard/pathfinder.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+- type: vessel
+  id: Pathfinder
+  name: NT Pathfinder
+  description: Once a scout ship serving with the Nanotrasen Marine Expeditionary Forces, this now decommissioned expedition capable ship can be yours!
+  price: 48920
```
Comment (Cheackraze) [inline Resources/Prototypes/_NF/Shipyard/pathfinder.yml:5]: if we could get it above 50k for an expedition vessel. Dont even have to add anything just make it over 50k just to get to a planet, the ship is pretty bare bones enough to make the other options worthwhile

**Comment by Kesiath** [comment]
increased price to 52k

**Comment by Cheackraze** [comment]
very empty, very entry level exped ship for those that want the most basic exped experience, perfect for 2-3 players.

**Review by Cheackraze — APPROVED** [review]

**Comment by EmmaFaith** [comment]
i think a "decommissioned military vessel" should at least have a single laser recharger

**Pull request merged**